packages/sit-onyx/src/components/OnyxMenuItem/types.ts

```diff
@@ -0,0 +1,20 @@
+import type { OnyxColor } from "../../types";
+
+export type OnyxMenuItemProps = {
+  /**
+   * URL that the menu item points to.
```

Comment:
We should also explain, that this property will switch between an anchor and a button element.